code/game/machinery/pipe/construction.dm

```diff
@@ -575,7 +575,7 @@
 	item_state = "buildpipe"
 	w_class = WEIGHT_CLASS_BULKY
 
-/obj/item/pipe_meter/attackby(obj/item/W as obj, mob/user as mob, params)
+/obj/item/pipe_meter/attackby(obj/item/W, mob/user, params)
```

Comment:
don't give this, but gotta love all the `/W`s taht exist because things used to be cast as `/obj/item/weapon`